src/authV2/azext_authV2/_help.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['webapp auth'] = """
+type: group
+short-summary: Manage webapp authentication and authorization in the v2 format.
+"""
+
+helps['webapp auth show'] = """
+type: command
+short-summary: Show the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Show the authentication settings for the webapp. (autogenerated)
+    text: az webapp auth show --name MyWebApp --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['webapp auth update'] = """
+type: command
+short-summary: Update the authentication settings for the webapp in the v2 format.
+examples:
+  - name: Update the client ID of the AAD provider already configured
+    text: >
+        az webapp auth update  -g myResourceGroup -n myUniqueApp --set identityProviders.azureActiveDirectory.registration.clientId=ecbacb08-df8b-450d-82b3-3fced03f2b27
+  - name: Pin the runtime version of the app to 1.4.7
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --runtime-version 1.4.7
+  - name: Configure the app with file based authentication by setting the config file path
+    text: >
+        az webapp auth update -g myResourceGroup -n myUniqueApp --config-file-path auth.json
```

Comment:
I wonder if people might mistake this as being a path to a local file (I assume it's supposed to be a server-side file path)... It might be more clear if you make it a fully-qualified path name.